### PR TITLE
Adding in an updated phpcs setup

### LINKED ISSUE #4: wpcom_vip_get_term_by is being deprecated.
https://github.com/alleyinteractive/logger/blob/dc3ee54f139c5253cf9e99002359a3133312807b/includes/class-ai-logger.php#L178

### PR DESCRIPTION
**Pull request by srtfisher**
- [x] Update phpcs
- [x] Make phpcs pass
- [x] Make travis pass

Merges in work from #5 which resolves #4, resolves #13

### CODE REVIEW

**Comment by szepeviktor** [comment]
@srtfisher Could you please apply the changes in https://github.com/alleyinteractive/logger/pull/5#issuecomment-507691675 ?

File: ai-logger.php [hunk 1]
```diff
@@ -25,9 +25,9 @@
 
 add_action(
 	'plugins_loaded',
-	function () { 
+	function () {
 		// Begin execution of the main plugin class.
-		(new AI_Logger_Plugin( __FILE__ ))->run();
+		( new AI_Logger_Plugin( __FILE__ ) )->run();
```
Comment (szepeviktor) [inline ai-logger.php:30]: What a mess.
Excuse me!

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      echo "xdebug.ini does not exist"
+    fi
+  - |
+    if [[ ! -z "$WP_VERSION" ]] ; then
+      bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
+      composer global require "phpunit/phpunit=4.8.*|5.7.*"
+    fi
+  - |
+    if [[ "$WP_TRAVISCI" == "phpcs" ]] ; then
+      composer global require wp-coding-standards/wpcs
```
[the remark this answers is not in the document]
Comment (srtfisher) [reply .travis.yml:33]: Probably not, will try!

[the remark this answers is not in the document]
Comment (srtfisher) [reply .travis.yml]: @joemcgill worked just fine, good call

**Pull request merged**